.pre-commit-config.yaml

```diff
@@ -80,13 +80,11 @@
         aiida/cmdline/utils/test_multiline.py|
         aiida/cmdline/utils/templates.py|
         aiida/cmdline/utils/echo.py|
-        aiida/control/.*.py|
         aiida/orm/data/cif.py|
         aiida/restapi/resources.py|
         aiida/restapi/translator/data/cif.py|
         aiida/utils/fixtures.py|
-        aiida/utils/fixtures.py|
-        docs/update_req_for_rtd.py|
+        docs/update_req_for_rtd.py
```

Comment:
Please add all the files to pre-commit, i.e. the test file and actual implementation

Reply:
aiida/backends/tests/cmdline/commands/test_graph.py and aiida/cmdline/commands/graph.py were already present in .pre-commit-config.yml so no additional change here.

Reply:
Yes but this is the PR for `group` not `graph`

Reply:
Right.. my mistake.